distributed/client.py

```diff
@@ -1325,17 +1325,19 @@ def running_workers(info):
                 ]
```

Comment:
FYI just above this line (cannot comment on the PR any better) we're filtering for `Status.running` workers, i.e. paused workers won't be detected here. This may be something that might hurt you in downscaling situations as well.

Reply:
Having the running filter is important because the scheduler knows about workers before they are up and running and this call should only unblock once the workers are actually reachable. I think this makes sense from a UX perspective and we also rely on this in tests.